File: durak-server-orleans/Durak.Server.API/Models/Room.cs
using Durak.Server.API.Enums;

namespace Durak.Server.API.Models;

public class Room(string connectionId, RoomSettings roomSettings, int playerCount)
{
    public string ConnectionId { get; set; } = connectionId;
    public Guid RoomId { get; set; } = roomSettings.Id;
    public Bet Bet { get; set; } = roomSettings.Bet;
    public int PlayerCount { get; set; } = playerCount;
}

File: durak-client-angular/src/app/models/Room.ts
export class Room {
    connectionId: string;
    roomId: string;
    playerCount: number;
    bet: number;

    constructor(connectionId: string, roomId: string, bet: number, playerCount: number) {
        this.connectionId = connectionId;
        this.roomId = roomId;
        this.playerCount = playerCount;
        this.bet = bet;
    }
}